subprojects/junit-jupiter/src/main/java/org/mockito/junit/jupiter/MockitoExtension.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2018 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.junit.jupiter;
+
+
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
+import org.junit.jupiter.api.extension.TestInstancePostProcessor;
+import org.mockito.Mockito;
+import org.mockito.MockitoSession;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.quality.Strictness;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.extension.ExtensionContext.Namespace.create;
+import static org.junit.platform.commons.support.AnnotationSupport.findAnnotation;
+
+/**
+ * Extension that initializes mocks and handles strict stubbings. This extension is the JUnit Jupiter equivalent
```

Comment:
What do you think about a new entry in main mockito javadoc in ```Mockito.java```? It's a noteworthy feature!!!

Reply:
Done :+1: